jax/_src/compiler.py

```diff
@@ -237,6 +240,28 @@ def backend_compile(
   # to take in `host_callbacks`
   return backend.compile(built_c, compile_options=options)
 
+def _total_graph_constant_size(operation: func.FuncOp) -> float:
```

Comment:
I'd be happier if this function had a unit test somewhere, since I'm not 100% sure it works correctly.